src/OmniSharp.Host/project.json

```diff
@@ -5,7 +5,6 @@
   },
   "dependencies": {
     "OmniSharp.Abstractions": "1.0.0",
-    "OmniSharp.Nuget": "1.0.0",
```

Comment:
oh god, no more trump references... I only hope he doesn't follow through repealing NAFTA (though looking at all his other promises... it might disappear!)